src/benchmark/processes.py

```diff
@@ -194,6 +198,85 @@ def _get_benchmark_app_metric(self, metric_name):
                     return None
 
 
+class OpenVINOBenchmarkCppProcess(OpenVINOProcess):
+    def __init__(self, test, executor, log, cpp_benchmark_path, perf_hint=''):
+        super().__init__(test, executor, log)
+        self._benchmark_path = cpp_benchmark_path
+        self._perf_hint = perf_hint
+
+    @staticmethod
+    def __add_perf_hint_for_cmd_line(command_line, perf_hint):
```

Comment:
A lot of duplication in OpenVINOBenchmarkCppProcess and OpenVINOBenchmarkPythonProcess, it's necessary to make parent class OpenVINOBenchmarkProcess

Reply:
Parent class exists now